### PR TITLE
Create Bringup README.md

### PR DESCRIPTION
**Pull request by yunlong-wang-cn**
add readme to bringup

### CODE REVIEW

**Review by berkgungor — CHANGES_REQUESTED** [review]

**Comment by ateRstones** [comment]
I think the pr #291 already implemented this

File: marimbabot_bringup/README.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# TAMS Master Project 2022/2023 - Bring up the system
+
+This is main entrance of the whole project, so you need two steps to bringup the whole system.
+
+1.  Launch the `marimbabot_ur5_bringup.launch` to bringup the serivce of the UR5 robot. **By launching this file, the robot's gripper could reinitialize itself, so what is holded by the gripper could fall down**
```
[the remark this answers is not in the document]
Comment (Flova) [reply marimbabot_bringup/README.md:5]: Maybe rename to driver instead of service

[the remark this answers is not in the document]
Comment (Flova) [reply marimbabot_bringup/README.md:5]: "what is holded by the gripper could fall down**" -> " anything held by the gripper is dropped!**"

**Review by Flova — APPROVED** [review]

**Comment by Flova** [comment]
> I think the pr #291 already implemented this

Oh you are correct. But we could still add the warning, detailing, that things might be dropped during startup.

**Review by Flova — CHANGES_REQUESTED** [review]
See the other comment

**Review by yunlong-wang-cn — COMMENTED** [review]
update doc

**Comment by yunlong-wang-cn** [comment]
resolve conflict and just add the warning from #291.

**Review by Flova — APPROVED** [review]

**Review by Flova — APPROVED** [review]

**Pull request merged**